packages/vscode-extension/src/qm/vsc_ui.ts

```diff
@@ -493,27 +494,44 @@ export class VsCodeUI implements UserInteraction {
     });
   }
 
-  async showMessage(
+  public async showMessage(
     level: "info" | "warn" | "error",
     message: string,
     modal: boolean,
     ...items: string[]
+  ): Promise<Result<string|undefined,FxError>>;
+
+  public async showMessage(
+    level: "info" | "warn" | "error",
+    message: Array<{content: string, color: Colors}>,
+    modal: boolean,
+    ...items: string[]
+  ): Promise<Result<string|undefined,FxError>>;
+
+  async showMessage(
```

Comment:
Do we still need the above two functions?

Reply:
This is the way to overload function, as chagong suggested.
![image](https://user-images.githubusercontent.com/71362691/122000658-c5838a80-cde1-11eb-8190-a77dce96ad07.png)